demos/ota/aws_iot_ota_update_demo.c

```diff
@@ -422,24 +437,40 @@ void vRunOTAUpdateDemo( bool awsIotMqttMode,
                            App_OTACompleteCallback,
                            ( TickType_t ) ~0 );
 
-            while( ( ( eState = OTA_GetAgentState() ) != eOTA_AgentState_Stopped ) && networkConnected )
+            while( ( ( eState = OTA_GetAgentState() ) != eOTA_AgentState_Stopped ) && _networkConnected )
             {
                 /* Wait forever for OTA traffic but allow other tasks to run and output statistics only once per second. */
-                vTaskDelay( OTA_DEMO_ONE_SECOND_DELAY );
+                IotClock_SleepMs( OTA_DEMO_TASK_DELAY * 1000 );
```

Comment:
This should be named `OTA_DEMO_TASK_DELAY_SECONDS`.  If a `#define` value is in units (time/distance/mass/etc.), include the unit in the name.

Reply:
Changed in https://github.com/aws/amazon-freertos/pull/1907/commits/181585836b9d05b07339d3c280ef512e4d85b85a